pkg/consts/qos.go

```diff
@@ -61,3 +63,24 @@ const (
 	PodAnnotationNetworkEnhancementAffinityRestricted     = "topology_affinity_restricted"
 	PodAnnotationNetworkEnhancementAffinityRestrictedTrue = "true"
 )
+
+// ResourcePluginPolicyName is a string type for QosResourceManager plugin policy
+type ResourcePluginPolicyName string
+
+// const variables for QRM plugin policy name
+const (
+	// ResourcePluginPolicyNameDynamic is the name of the dynamic policy.
+	ResourcePluginPolicyNameDynamic ResourcePluginPolicyName = "dynamic"
+	// ResourcePluginPolicyNameNative is the name of the native policy.
+	ResourcePluginPolicyNameNative ResourcePluginPolicyName = "native"
+	// ResourcePluginPolicyNameStatic is the name of the static policy.
+	ResourcePluginPolicyNameStatic ResourcePluginPolicyName = "static"
```

Comment:
Do we need this `static` policy?

Reply:
these consts are referenced by QRM plugins in `katalyst-core`  to ensure the consistency, so static policy which is used in io and network plugin is also defined here